n_motors: 4
mass: 3.5 # [kg]
arm_length: 0.25 # [m]
body_height: 0.1 # [m]
motor_time_constant: 0.03
g: 9.81 # [ms^-2]

propulsion:

  force_constant: 0.00000022970
  moment_constant: 0.07

  prop_radius: 0.15 # [m]

  # [moment;
  #  total_force] = matrix * motor_force
  mixing_matrix: [
    0,  0, 1,  -1,
    -1, 1, 0,  0,
    1,  1, -1, -1,
    1,  1, 1,  1,
  ]

rpm:
  min: 0
  max: 8951
